src/commcare_cloud/commands/ansible/service.py

```diff
@@ -274,10 +275,60 @@ class Nginx(AnsibleService):
     inventory_groups = ['proxy']
 
 
-class Elasticsearch(AnsibleService):
+class ElasticsearchClassic(AnsibleService):
+    name = 'elasticsearch-classic'
+    service_name = 'elasticsearch'
+    inventory_groups = ['elasticsearch']
+
+
+class Elasticsearch(ServiceBase):
     name = 'elasticsearch'
+    service_name = 'elasticsearch'
     inventory_groups = ['elasticsearch']
 
+    def execute_action(self, action, host_pattern=None, process_pattern=None):
+        if action == 'status':
+            return ElasticsearchClassic(self.environment, self.ansible_context).execute_action(action, host_pattern, process_pattern)
+        else:
+            response = six.moves.input("This function does more than stop and start the elasticsearch service. For that, use elasticsearch-classic."
+                                       "\nStop will: stop pillows, stop es, and kill -9 if any processes still exist after a period of time. "
+                                       "\nStart will start pillows and start elasticsearch "
+                                       "\nRestart is a stop followed by a start.\n Continue? [y/n]: ")
+            if response.lower() == "n":
```

Comment:
I'd use `ask` for this the way other commands in commcare-cloud do for a consistent experience (and consistent code). E.g. https://github.com/dimagi/commcare-cloud/blob/4a61f5978c8391bea1287ad6f34f478ca6127e24/src/commcare_cloud/commands/ansible/ansible_playbook.py#L229. (`ask` is just a little utility function we wrote, not a common python thing.)

Reply:
Good to know
https://github.com/dimagi/commcare-cloud/pull/1990/commits/bed03233686c862186427cafe6a71b06d95705c2